local-cli/runIOS/runIOS.js

```diff
@@ -82,20 +81,44 @@ function _runIOS(argv, config, resolve, reject) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
 
-  const appPath = `build/Build/Products/Debug-iphonesimulator/${inferredSchemeName}.app`;
-  console.log(`Installing ${appPath}`);
-  child_process.spawnSync('xcrun', ['simctl', 'install', 'booted', appPath], {stdio: 'inherit'});
+  var appPath = `build/Build/Products/Debug-iphonesimulator/${inferredSchemeName}.app`;
+  const xcodeBuildProcess = child_process.spawn('xcodebuild', xcodebuildArgs);
+
+  xcodeBuildProcess.stdout.on('data', (data) => {
+    console.log(data.toString());
```

Comment:
You're right. Good catch, didn't even notice the unnecessary newlines in this huge log :-) I will fix it